pyeudiw/satosa/backend.py

```diff
@@ -206,6 +206,8 @@ def pre_request_endpoint(self, context: Context, internal_request, **kwargs) ->
         result = self.template.qrcode_page.render(
             {
                 "qrcode_color": self.config["qrcode"]["color"],
+                "qrcode_size": self.config["qrcode"]["size"],
+                "qrcode_logo_path": self.config["qrcode"]["logo_path"],
                 "qrcode_text": res_url,
                 "qrcode_expiration_time": self.config["qrcode"]["expiration_time"],
                 "state": state,
```

Comment:
where is "static"?

https://github.com/italia/eudi-wallet-it-python/pull/230/files#diff-cca91a28aa9487e7894dee05653610c0ee47c14ba8cb54dd594ef898b0154c04R13

Reply:
@Gartic99 you told me that this is loaded in the default context of the jinja2 template processor, that's good.